src/main/java/service/impl/InputHandler.java

```diff
@@ -0,0 +1,32 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import service.InputFileHandler;
+
+public class InputHandler implements InputFileHandler {
+    @Override
+    public List<String> readFile(String fileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(
+                new FileReader(fileName))) {
+            StringBuilder fileDataCollector = new StringBuilder();
+            String line = bufferedReader.readLine();
+
+            while (line != null) {
+                fileDataCollector.append(line).append(System.lineSeparator());
+                line = bufferedReader.readLine();
+            }
+
```

Comment:
You have a redundant empty line here. Remember, according to the checklist, class or method implementation should not begin with an empty line.